src/Behavior/DefaultMove.h

```diff
@@ -67,13 +67,13 @@ class DefaultMove : public Movable
 
 
 template<class T>
-Vector2 DefaultMove<T>::GetChildrenTranslate() const
+glm::vec2 DefaultMove<T>::GetChildrenTranslate() const
 {
 	return childrenTranslate;
 }
 
 template<class T>
-void DefaultMove<T>::TranslateChildren(Vector2 translate)
+void DefaultMove<T>::TranslateChildren(const glm::vec2 &translate)
```

Comment:
Original parameter was without a reference. Please remove the reference and pass by copy.